scandium-core/src/main/java/org/eclipse/californium/scandium/DTLSConnector.java

```diff
@@ -491,8 +494,8 @@ public final synchronized void destroy() {
 
 	private void receiveNextDatagramFromNetwork() throws IOException {
 
```

Comment:
I would prefer to limit the access scope of the `receiverBuffer`.
Could we move that field to the `Worker `and pass it in as argument to `receiveNextDatagramFromNetwork`?
May be we can even reuse the `DatagramPacket` and pass that to `receiveNextDatagramFromNetwork`?